app/ui-message/client/messageBox.js

```diff
@@ -146,6 +146,10 @@ Template.messageBox.onCreated(function() {
 	this.isMicrophoneDenied = new ReactiveVar(true);
 	this.sendIconDisabled = new ReactiveVar(false);
 	messageBox.emit('created', this);
+	window.addEventListener('keydown',function(e){
```

Comment:
I'm not sure this is the best place to do this code. @ggazzo / @tassoevan can you please advise the best place to do this?

@mohamedar97 also this code is not following our code style. please run `meteor npm run lint` before pushing your changes.